### ISSUE TITLE
TSvg Load/SaveToStream inefficient

### ISSUE THREAD

**Issue by pyscripter** [EtheaDev/SVGIconImageList#19]
The TSvg LoadFromStream, SaveToStream convert text to a TStringList and then load./save that.  This is inefficient.  Replace with the following:

```pascal
procedure TSVG.LoadFromStream(Stream: TStream);
var
  Size: Integer;
  Buffer: TBytes;
begin
  Stream.Position := 0;
  Size := Stream.Size;
  SetLength(Buffer, Size);
  Stream.Read(Buffer, 0, Size);
  LoadFromText(TEncoding.UTF8.GetString(Buffer));
end;

procedure TSVG.SaveToStream(Stream: TStream);
var
  Buffer: TBytes;
begin
  Buffer := TEncoding.UTF8.GetBytes(FSource);
  Stream.WriteBuffer(Buffer, Length(Buffer));
end;
```

The above deals with #18

**Comment by carloBarazzetta** [comment]
Ok.

**Comment by carloBarazzetta** [comment]
Resolved into 1.7 version.
